bin/setup

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env ruby
 require 'pathname'
 
+def system(*args)
+  puts args.first
+  super
+end
```

Comment:
Yeah, exactly, that's the idea, to at least know what we're running, instead of only the output of the command.